### PR TITLE
Remove the unused property minLevel from CategoryConfig

### PR DESCRIPTION
**Pull request by debae**
This change was already part of PR https://github.com/quarkusio/quarkus/pull/5861 but is a separate one as proposed by @dmlloyd .

### CODE REVIEW

**Review by machi1990 — APPROVED** [review]
LGTM, thanks @debae 

Of course let's wait for @dmlloyd approval.

**Review by dmlloyd — APPROVED** [review]

**Pull request merged**
